terraform/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_instance" "web-srv" {
+  ami                    = data.aws_ami.web_ami.id
+  instance_type          = var.inst_type
+  vpc_security_group_ids = [aws_security_group.sg_app.id, aws_security_group.sg_db.id]
+  key_name               = "aws"
```

Comment:
`key_name` has a hardcoded value.
It's OK in this case, but it can be parametrized, so the code can be reused in different scenarios.